test/helper.rb

```diff
@@ -58,7 +58,7 @@ def api
   # Ignore transient params when building VCR fixtures.
   matcher = VCR.request_matchers.uri_without_param(
     'AWSAccessKeyId', 'SellerId', 'Signature', 'Timestamp', 'StartDate',
-    'CreatedAfter', 'QueryStartDateTime'
+    'CreatedAfter', 'QueryStartDateTime', 'Version'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.